packages/components/ovh-shell/README.md

```diff
@@ -1 +1,178 @@
-# ovh-shell
+# @ovh-ux/shell
+
+The goal of this lib is to provide utilities to communicate with a Shell. It exposes an API for that purpose, it also exposes a list of Plugins you can use with the Shell.
+
+## Table of contents
+
+- [Lib usage](#usage)
+- [API Docs](#api)
+  - [Shell methods](#shell)
+    - [initShell](#initShell)
+  - [Plugins API](#plugins)
+    - [i18n](#i18n)
+    - [routing](#routing)
+    - [auth](#auth)
+    - [ux](#ux)
+    - [tracking](#tracking)
+    - [navigation](#navigation)
+
+## Usage
+
+To use this lib, you have to add `@ovh-ux/shell` to your dependencies and then build the package under `packages/components/ovh-shell/`.
+
+The lib is fully tree-shaked, so to import something you just have to import it as an esModule :
+
+```ts
+import { shell } from '@ovh-ux/shell';
+```
+
+## API
+
+### Shell
+
+#### `initShell`
+
+This method initializes a Shell given an iframe. You can initiate a Shell without an iframe, but keep in mind that the communications will use the `window` object instead of the `iframe.contentWindow` object.
+
+_Example :_
+
+```ts
+import { shell } from '@ovh-ux/shell';
+
+const shellApi = shell.initShell(yourIframe);
+```
+
+This method inits a shell and returns an API that allows you to communicate with it.
+
+The returned object contains the following methods :
+
+- `emitEvent`(`(eventId: string, data: unknown) => void`) : This method allows to emit a custom event from the Shell.
```

Comment:
It is unclear to me how we can use this.

What kind of event can we pass on, and where are the event listeners defined at? Do we have to create a listener somewhere or are there a number of listener in the shell?

Also, would it be better practice to prefix our eventId with our app name to avoid mix-ups (e.g. `my-app:my-event`)? If so it should be told (or we should have an exhaustive list of events naturally listened to by the Shell).

An example could help.